submissions/7SergSmith7/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>HTML-CSS-Popup</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <nav class="right_navigation">
+          <a href="#" class="right_navigation__mail">Mail</a>
+          <a href="#" class="right_navigation__images">Images</a>
+          <input type="checkbox" id="popup-check" class="popup_btn" />
+          <label class="popup_btn-icon" tabindex="0" for="popup-check"></label>
+          <div class="popup">
+            <div class="popup-content">
+              <div class="main-menu">
+                <a href="#" class="popup-item">
+                  <figure class="popup-item-icon">
+                    <img src="./img/maps.png" alt="maps" />
+                    <figcaption class="item-title">Maps</figcaption>
```

Comment:
Why this class looks different from parent elements hierarchy:
```
HTML
popup-item > popup-item-icon > item-title
```

Reply:
Is it critical?

Reply:
It's not critical for this task, but in real project can be a couple of questions to you).
One good example of naming and creating hierarchy of CSS is BEM methodology.[ Read more about it](https://ru.bem.info/methodology/quick-start/)